wpeverest/everest-forms
# Fix - Email template on email clone, save and continue forms

### All Submissions:


### Changes proposed in this Pull Request:
Previously, the email clone was not working and along that the email template was not working on the Save and continue Form. This PR fixes these mentioned issues.

Closes # .

### How to test the changes in this Pull Request:

1. Create a form and duplicate the existing email. Check whether the accurate email is being sent or not according to the content and the template chosen.
2. Enable the **Save and Continue** from the dashboard
3. Go to **Form Settings>Save and Continue**> Enable toggle for Save and Continue.
4. Now, go to **Form Settings>Emails>Enable Save and Continue**.
5. Choose the email template for **Save and Continue**.
6. Check whether the accurate email is sent accurately according to the email template and its content or not.

### Types of changes:



### Other information:


### Changelog entry

> Fix - Email template on email clone, save and continue forms
